src/Amalgam/SBFDSColumnData.h

```diff
@@ -221,28 +218,9 @@ class SBFDSColumnData
 			{
 				double old_number_value = GetResolvedValue(old_value_type, old_value).number;
 				double new_number_value = GetResolvedValue(new_value_type, new_value).number;
-				if(EqualIncludingNaN(old_number_value, new_number_value))
+				if(old_number_value == new_number_value)
```

Comment:
you seem to have removed all usages of EqualIncludingNaN() except from line 1066 of SBFDS.cpp.  since that's its only use, maybe copy-paste that one liner directly into that cpp and get rid of that function?